docs/Data/Validation/constructors/Valid.md

```diff
@@ -0,0 +1,23 @@
+# `Validation.Valid<F, S>(S value)`
```

Comment:
Consider updating the generic type abbreviations _throughout_. (F and S I assume meant Failure and Success.)

Reply:
I updated them in a force push with `<I, V>`.